src/%IPM/Repo/Remote/PackageService.cls

```diff
@@ -20,6 +22,14 @@ Method GetInfo(url As %String) As %DynamicObject
 	$$$ThrowStatus($$$ERROR($$$GeneralError,"Registry server not available."))
 }
 
+Method GetLatestIPMVersion() As %String
```

Comment:
Nit: These methods should not be for a specific module ideally so perhaps just change it to `GetLatestModuleVersion` and pass as an argument the module name